development/development/coding_guidelines.rst

```diff
@@ -5,8 +5,8 @@ phpBB's coding guidelines are maintained in its
 `source code repository <https://github.com/phpbb/phpbb>`_. You can find the
 latest versions on area51:
 
-* Rules for `Olympus (3.0.x) code <http://area51.phpbb.com/docs/30x/coding-guidelines.html>`_
-* Rules for `Ascraeus (3.1.x) code <http://area51.phpbb.com/docs/31x/coding-guidelines.html>`_
+* Rules for `Olympus (3.0.x) code <https://area51.phpbb.com/docs/30x/coding-guidelines.html>`_
+* Rules for `Ascraeus (3.1.x) code <https://area51.phpbb.com/docs/31x/coding-guidelines.html>`_
```

Comment:
Maybe the right time to remove 3.0 and 3.1 and add 3.2 and 3.3. instead?

Reply:
Well, this PR is still agains 3.1.x so maybe as another task after merging this